source/manual/assets.html.md

```diff
@@ -44,27 +44,24 @@ There are currently two systems for uploading, managing and serving
 user-supplied assets on GOV.UK.
 
 Asset Manager is an API that is called internally by Publisher, Specialist
-Publisher, Manuals Publisher and Travel Advice Publisher to manage their
+Publisher, Manuals Publisher, Travel Advice Publisher and Whitehall to manage their
 uploads. It serves the uploaded assets on assets.publishing.service.gov.uk/media
 
-Whitehall is a standalone publishing app that manages its own uploaded assets.
-It also serves those assets, via both
-assets.publishing.service.gov.uk/government/uploads and
-www.gov.uk/government/uploads, even for content that has been migrated to the
+Whitehall is a standalone publishing app that manages a type of asset called "attachments". It serves attachments, via both
+`assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data` and
+`www.gov.uk/government/uploads/government/uploads/system/uploads/attachment_data`, even for content that has been migrated to the
```

Comment:
Since https://github.com/alphagov/whitehall/pull/3918 requests for attachments on `www.gov.uk` are now redirected to `assets.publishing.service.gov.uk`, so it might be simpler not to mention `www.gov.uk` here...?